internal/service/remotecfg/remotecfg.go

```diff
@@ -131,7 +131,7 @@ func New(opts Options) (*Service, error) {
 
 	return &Service{
 		opts:   opts,
-		ticker: time.NewTicker(math.MaxInt64),
+		ticker: jitter.NewTicker(math.MaxInt64-100*time.Millisecond, 100*time.Millisecond),
```

Comment:
This was a bit confusing in that we are creating a super long ticker to start but I see it mirrors the old implementation. I assume this prevents the service from doing anything in the case it isn't actually in use. 

nit: Can the first argument be just `math.MaxInt64`?

Reply:
Yes; it's the easiest way I found to not have two separate code paths; have a ticker that is actually going to tick after `219250468-12-04 15:30:07 +0000 UTC` which I think is a long way into the future 😅.

The reason why the first argument is not just MaxInt64, is that this implementation chooses a jitter value in the `[d-j, d+j)` interval, and that would lead in an overflow and make the ticker panic.